### DISCUSSION TITLE
[ FF7 ] I can't load a save file. Game crashes

### DISCUSSION THREAD

**Discussion by ghost** [julianxhokaxhiu/FFNx#177] [General]
When i try to load a save file a window pops up saying:

"Oops! Something very bad happened.

Wrote emergency save to save/crash.ff7 dir.

Please provide a copy of those files along with FFNx.LOG
when reporting this error at 
https://github.com/julianxhokaxhiu/FFNx/issues"

Help me please. I'm frustrated.

**Comment by eve-atum** [comment]
Make sure the "60fps" option in the Animations mod is disabled (or better yet, get rid of the animations mod entirely).  Past that, we can't suggest anything without seeing the log.

**Comment by ghost** [comment]
> Make sure the "60fps" option in the Animations mod is disabled (or better yet, get rid of the animations mod entirely). Past that, we can't suggest anything without seeing the log.

The animations mod was the problem indeed. Thank you! How can i show you the log?

**Comment by eve-atum** [comment]
Well, since we've identified that the animations mod was the problem, there's no need for the log ;)  Enjoy the game!

**Comment by ghost** [comment]
> Well, since we've identified that the animations mod was the problem, there's no need for the log ;) Enjoy the game!

The problem came back. This time when i press play in 7th heaven that window shows up right away. I deleted the animations mod but this doesn't seem the cause of the problem anymore :(

I found out that the solution is Disabling Windows DEP. If you disable it even the animations mod will work (also with 60 fps). If anyone is having the same problem here's the guide to do it https://kb.omni-ts.com/entry/55/

**Comment by julianxhokaxhiu** [comment]
Please post and talk about 7th issues on the forum, this repository is not related to 7th ( despite 7th using FFNx ). Thank you :)
